### ISSUE TITLE
Video data

### ISSUE THREAD

**Issue by DrRonne** [elmoiv/redvid#9]
Hello, thank you for making this library, it is awesome and works great!

One addition I would suggest, is to make something that can check video data before you download the video. For example, I only want to download videos that are shorter than one minute. Right now I would have to download it and check the time afterwards. It would be cool if I could check this beforehand.

**Comment by elmoiv** [comment]
Thanks @DrRonne for your Ideas. I will consider these features in my next update.
Stay tuned!

**Comment by bcornw2** [comment]
if you can afford the temporary disk space and the runtime speed, you can do what I do and check the size of the file after download, then delete it. I know its not optimal but it was only like 2 extra loc after the initial call.

**Comment by elmoiv** [comment]
I found a workaround for viewing video size and duration before downloading.
I am implementing it now so you can decide a maximum size or duration so if the targeted video quality does not meet the requirements passed, It will return with error number instead.

Here is a piece of my current work:
```python
...
        self.duration = getDuration(self.page)
        self.size = int(
                    self.head(
                                self.video,
                                _proxies=self.proxies
                            ).headers['Content-Length']
                        )
...
```

Stay tuned :)

**Comment by elmoiv** [comment]
Published v1.0.9:
  * Added the ability to disable logging.
  * Maximum video size can be set.
  * Maximum video duration can be set.

To download videos shorter than 1 minute do the following:
```python
from redvid import Downloader

reddit = Downloader()
reddit.max_d = 60  # 60 seconds
reddit.url = 'https://v.redd.it/c8oic7ppc2751'
reddit.download()
```

Now if the video exceeds 1 minutes, nothing will be downloaded and function will return error number corresponding to duration which equals 1.
